types/react-jsonschema-form/index.d.ts

```diff
@@ -256,16 +256,19 @@ declare module 'react-jsonschema-form/lib/components/fields/SchemaField' {
     import { JSONSchema6 } from 'json-schema';
     import { FieldProps, UiSchema, IdSchema, FormValidation } from 'react-jsonschema-form';
 
-    export interface SchemaFieldProps<T = any> {
-      schema: JSONSchema6;
-      uiSchema: UiSchema;
-      idSchema: IdSchema;
-      formData: T;
-      errorSchema: FormValidation;
-      registry: FieldProps['registry'];
-    }
-
-    export class SchemaField<T> extends React.Component<SchemaFieldProps<T>> {}
+    export type SchemaFieldProps<T = any> = Pick<
+      FieldProps<T>,
+      | 'schema'
+      | 'uiSchema'
+      | 'idSchema'
+      | 'formData'
+      | 'errorSchema'
+      | 'registry'
+    >;
+
+    export default class SchemaField extends React.Component<
```

Comment:
Why use `export default` for `SchemaField`?

Reply:
Inside `react-jsonschema-form/src/components/fields/SchemaField.js`:
```js
class SchemaField extends React.Component {
    ...
}

...

export default SchemaField;
```

 SchemaField is exported as `default`. When we import from that file location, we can only pull the `default` since we don't export the class as well.